src/styles/block/header-style.scss

```diff
@@ -0,0 +1,87 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+body {
+  padding: 0;
+  margin: 0 auto;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+  position: relative;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  text-align: center;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  list-style-type: none;
+  display: inline-block;
+}
+
+.nav__link {
+  padding: 0;
+  line-height: 60px;
+  font-size: 12px;
+  font-weight: 500;
+  align-items: center;
+  display: inline-block;
+  position: relative;
+  text-decoration: none;
+  color: #000;
+  margin-left: 20px;
+  text-transform: uppercase;
+  height: 60px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+  margin-left: 0;
```

Comment:
The `.is-active` class should not reset the `margin-left` to 0. Instead, use the `:not(:first-child)` selector on `.nav__link` to apply `margin-left` only when it's not the first child.